cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/service/IndyService.kt

```diff
@@ -52,7 +55,15 @@ class IndyService(services: AppServiceHub) : SingletonSerializeAsToken() {
         val tailsPath = "tails"
         val didConfig = DidConfig(did, seed, null, null)
 
-        val walletUser = IndySDKWalletUser(wallet, didConfig, tailsPath)
+        val walletUser = if (did != null && wallet.getOwnIdentities().map { it.did }.contains(did))
```

Comment:
I know, you like it, but this if statement will actually be shorter without `also`

Reply:
any proofs?)